src/RegistryImportFeed.php

```diff
@@ -49,6 +64,79 @@ public function getLatest()
         );
     }
 
+    /**
+     * Set the handler used to manipulate the filesystem
+     *
+     * @param GeneratedAssetHandler $handler
+     * @return $this
+     */
+    public function setAssetHandler(GeneratedAssetHandler $handler)
+    {
+        $this->assetHandler = $handler;
+        return $this;
+    }
+
+    /**
+     * Get the handler used to manipulate the filesystem, and add the ListFiles plugin from Flysystem to inspect
+     * the contents of a directory
+     *
+     * @return GeneratedAssetHandler
+     */
+    public function getAssetHandler()
+    {
+        $this->assetHandler->getFilesystem()->addPlugin(new ListFiles);
```

Comment:
Do this on setter (called once) and not on the getter (called many times).

Reply:
Hah, yeah true